timed/tracking/views.py

```diff
@@ -267,8 +267,7 @@ def bulk(self, request):
         if "task" in fields:
             # unreject report if task has changed
             fields["rejected"] = False
-            if fields["task"].project.billed:
-                fields["billed"] = fields["task"].project.billed
+            fields["billed"] = True if fields["task"].project.billed else False
```

Comment:
```suggestion
            fields["billed"] = fields["task"].project.billed
```

this should also work since its a boolean

Reply:
@c0rydoras that's where this started off at https://github.com/adfinis/timed-backend/pull/988#discussion_r1507771956

Reply:
```py
fields["billed"] = bool(fields["task"].project.billed)
```

Reply:
if it isn't a boolean this accomplishes the same